third_party/amd/include/TritonAMDGPUTransforms/Passes.td

```diff
@@ -124,4 +124,17 @@ def TritonAMDGPUConvertToBufferOps : Pass<"tritonamdgpu-convert-buffer-ops", "ml
   let dependentDialects = ["mlir::triton::amdgpu::TritonAMDGPUDialect"];
 }
 
+def TritonAMDGPUBlockPingpong: Pass<"tritonamdgpu-block-pingpong", "mlir::ModuleOp"> {
+  let summary = "Transform into 2 block ping-pong";
```

Comment:
"Interleaving instructions from two warps on the same SIMD to better utilize matrix core"

Reply:
Done